java-generator/core/src/main/java/io/fabric8/java/generator/Config.java

```diff
@@ -64,60 +66,6 @@ public Config(
     }
   }
 
-  public Config(
```

Comment:
low risk, but this is a breaking change, should we go through a round of deprecation before removing those?